test/e2e/resourceinterpreter_test.go

```diff
@@ -422,4 +422,208 @@ end`,
 			}
 		})
 	})
+
+	ginkgo.Context("InterpreterOperation Retain testing", func() {
+		var waitTime time.Duration
+		var updatedPaused bool
+
+		ginkgo.BeforeEach(func() {
+			waitTime = 5 * time.Second
+			updatedPaused = true
+			customization = testhelper.NewResourceInterpreterCustomization(
```

Comment:
Shall we clean the customization after test?

Reply:
We clean the customization at here:
https://github.com/karmada-io/karmada/blob/master/test/e2e/resourceinterpreter_test.go#L314